images/nginx/fastcgi.conf

```diff
@@ -13,6 +13,9 @@ set_by_lua_block $remote_addr_clean {
   end
 }
 
+# Allow downstreams to add fastcgi_params.
```

Comment:
It may be worth mentioning in this comment that *usually* any values included this way may be overwritten by the default values below (but there may be FastCGI processors that don't handle them in strict order)